### PR TITLE
docs: consolidated client creation snippets

### PR DESCRIPTION
**Pull request by craiglabenz**
This comes as part of an effort to consolidate the [`firestore_setup_client_create`](https://cloud.google.com/firestore/docs/samples/firestore-setup-client-create) and [`firestore_setup_client_create_with_project_id`](https://cloud.google.com/firestore/docs/samples/firestore-setup-client-create-with-project-id) regions.

### CODE REVIEW

**Comment by craiglabenz** [comment]
I am not sure why, but grpc's PHP libraries are reluctant to install on my machine - meaning I have not yet figured out how to run these tests locally. Any tips from @bshaffer or @dazuma would be great!

**Comment by bshaffer** [comment]
@craiglabenz have you tried following the installation instructions for gRPC at https://cloud.google.com/php/grpc ?

File: firestore/test/firestoreTest.php [hunk 1]
```diff
@@ -74,13 +74,15 @@ public static function tearDownAfterClass(): void
...
+        $output = self::runFunctionSnippet('setup_client_create', []);
         $this->assertStringContainsString('Created Cloud Firestore client with default project ID.', $output);
     }
 
     public function testInitializeProjectId()
     {
-        $output = $this->runFirestoreSnippet('setup_client_create_with_project_id');
+        # The lack of a second parameter implies that a non-empty projectId is
+        # supplied to the snippet's function.
+        $output = $this->runFirestoreSnippet('setup_client_create');
```
Comment (bshaffer) [inline firestore/test/firestoreTest.php:85]:
```suggestion
        $output = $this->runFirestoreSnippet('setup_client_create', [self::$projectId]);
```

File: firestore/test/firestoreTest.php [hunk 2]
```diff
@@ -74,13 +74,15 @@ public static function tearDownAfterClass(): void
 
     public function testInitialize()
     {
-        $output = $this->runFirestoreSnippet('setup_client_create', []);
+        $output = self::runFunctionSnippet('setup_client_create', [self::$projectId]);
         $this->assertStringContainsString('Created Cloud Firestore client with default project ID.', $output);
```
Comment (bshaffer) [inline firestore/test/firestoreTest.php:78]: This test wasn't updated
```suggestion
        $this->assertStringContainsString('Created Cloud Firestore client with project ID: ', $output);
```

**Pull request merged**